CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+### 1.15.0
+- Fix: Update telemetry to 0.8.0 [#1023](https://github.com/redhat-developer/vscode-yaml/pull/1023)
```

Comment:
Consider adding:
- Your version bump to testing related dependencies
- Gorkem's docs fix: https://github.com/redhat-developer/vscode-yaml/commit/1e22ef376bdaa21e27b9c13c9729f1bed731337e